sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -138,7 +137,18 @@ export interface CertificateOperationError {
 }
 
 // @public
-export interface CertificatePolicy {
+export type CertificatePolicy = CertificatePolicyProperties & RequireAtLeastOne<PolicySubjectProperties>;
+
+// @public
+export module CertificatePolicy {
```

Comment:
TypeScript modules, like namespaces, should be avoided. Would something like the following work?

```
export const CertificatePolicy = { ... }?
```

Reply:
It will definitely work. I'll update it.

Reply:
Pushed a fix!